src/pages/LoginPage.js

```diff
@@ -1,33 +1,63 @@
 import React, { useState } from 'react';
-import { NavLink } from "react-router-dom";
+import { NavLink, useNavigate } from "react-router-dom";
 import Logo from "../assets/images/PeakPulse_white.png";
-import PrimaryButton from "../components/common/PrimaryButton";
 import image from "../assets/images/CityOfRocks_KieranHadley_BathRock_Calamari12b-2048x1152.jpg"
+import Parse from 'parse/dist/parse.min.js';
+import { Input, Button } from 'antd';
 
-function LoginPage() {
-    const [loginData, setLoginData] = useState({
-        email: '',
-        password: '',
-    });
+export const LoginPage = () => {
+    // State variables
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [currentUser, setCurrentUser] = useState(null);
+    const [errorMessage, setErrorMessage] = useState('');
 
-    const handleInputChange = (e) => {
-        const { name, value } = e.target;
-        setLoginData({ ...loginData, [name]: value });
+    const navigate = useNavigate();
+
+    const doUserLogIn = async function () {
+        // Note that these values come from state variables that we've declared before
+        const usernameValue = username;
+        const passwordValue = password;
+        try {
+            const loggedInUser = await Parse.User.logIn(usernameValue, passwordValue);
+            // To verify that this is in fact the current user, `current` can be used
+            const currentUser = await Parse.User.current();
+            console.log(loggedInUser === currentUser);
+            // Clear input fields
+            setUsername('');
+            setPassword('');
+            // Update state variable holding current user
+            getCurrentUser();
```

Comment:
you're calling `Parse.User.current()` also a few lines above. and also inside of this. why not do a simple set here, and be done with it?

Reply:
Good point! I've changed the code, so we only have one function setting the current user.